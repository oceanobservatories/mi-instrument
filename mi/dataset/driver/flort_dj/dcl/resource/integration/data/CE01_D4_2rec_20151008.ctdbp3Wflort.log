2015/10/08 17:48:40.005 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/10/08 18:22:20.688 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/10/08 18:29:14.783 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/10/08 18:29:19.690 [ctdbp3:DLOGP6]: 16.0207,  0.00003,   -0.070, 50, 53, 874, 08 Oct 2015 18:29:15
2015/10/08 18:31:03.710 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/10/08 19:29:14.074 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/10/08 19:29:18.989 [ctdbp3:DLOGP6]: 11.8015,  3.77270,    0.896, 4130, 2360, 3627, 08 Oct 2015 19:29:14
2015/10/08 19:31:04.973 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/10/08 20:29:04.689 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/10/08 20:29:18.723 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/10/08 20:31:04.418 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/10/08 21:29:04.940 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/10/08 21:29:20.984 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/10/08 21:31:04.378 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/10/08 22:29:06.073 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/10/08 22:29:22.106 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/10/08 22:31:05.471 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/10/08 23:29:03.868 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/10/08 23:29:17.902 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/10/08 23:31:04.738 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
