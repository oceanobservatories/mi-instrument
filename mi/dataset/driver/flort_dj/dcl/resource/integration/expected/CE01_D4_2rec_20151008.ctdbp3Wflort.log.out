
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE01ISSM | SBD17 | 06-FLORTD000 | 3653251200 | telemetered | 3653317759.6900000572 |          4 |         3653317755 | 3653317759.6900000572 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |              50 |             874 |             53 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3653251200 | telemetered | 3653321358.9890003204 |          4 |         3653321354 | 3653321358.9890003204 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            4130 |            3627 |           2360 |        null

(2 rows)
